### DISCUSSION TITLE
Is jotai just for React ?

### DISCUSSION THREAD

**Discussion by RayyanNafees** [pmndrs/jotai#2045] [General] [answered]
How to use Jotai with vanilla js

Like your zustand has that feature, why not Jotai?

**Comment by dai-shi** [comment, marked as answer]
Same as Zustand, Jotai can be used in Vanilla JS.
There are third-party libraries for Solid and Vue bindings.

**Reply by loganvolkers** [reply]
I just recently contributed to improving the [Vue bindings](https://github.com/wobsoriano/jotai-vue), and I can confirm that there is nothing super tied to React.

**Comment by dai-shi** [comment]
See also #1811
